haphilipsjs/__init__.py

```diff
@@ -73,7 +73,7 @@ def getAudiodata(self):
         if audiodata:
             self.min_volume = int(audiodata['min'])
             self.max_volume = int(audiodata['max'])
-            self.volume = audiodata['current']
+            self.volume = audiodata['current'] / self.max_volume
```

Comment:
Wouldn't this break the current implementation for TVs that are pre-5?

Reply:
Ok. Thought I had already implemented setting the volume back then. But if it wasn't available in the first place it can't break anyways.